modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying architectures on the Agent-based installer 
+
+You can check which architecture your {product-title} agent installer cluster is using by viewing the version binary.
+
+.Prerequisites: 
+
+* You installed the OpenShift CLI (oc)
+* You have the openshift-installer binary
+
+.Procedure: 
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshfit-install 4.15.z
+built from commit e80d50ac8c9ab65833045d648a01eb1a0a3a6d7d
+release image quay.io/openshift-release-dev/ocp-release@sha256:27d41f99e0784a2f596e773799983147b794df48d1f7536d51a95a60945b1931
+release architecture amd64
+----
++
+If you are using the `amd64` or `multi` installer payload, the `release architecture` output of this command displays to `amd64` by default
```

Comment:
```suggestion
If you are using the `multi` installer payload, the `release architecture` output of this command displays to the architecture of the host you are running the installer in.
```

Reply:
I changed it just a touch to make it more direct and avoid the preposition at the end